### PR TITLE
NMS-13642: Add breaking changes for Twin

### PR DESCRIPTION
**Pull request by cgorantla**
### External References

* JIRA (Issue Tracker): http://issues.opennms.org/browse/NMS-13642

### CODE REVIEW

File: docs/modules/releasenotes/pages/whatsnew.adoc [hunk 1]
```diff
@@ -29,6 +29,12 @@ If you wish to revert to the previous behavior, create an `$OPENNMS_HOME/etc/ope
 RUNAS=root
 ----
 
+=== Additional config needed when using Kafka for Minion Communication.
+
+Minion no longer uses OpenNMS Rest API to get Trapd Config updates. It uses Twin mechanism to sync with Trapd config.
+Minion needs additional config both on Minion and OpenNMS to enable Twin. Refer to deployment section from the docs.
```
Comment (mmahacek) [inline docs/modules/releasenotes/pages/whatsnew.adoc:35]:
```suggestion
The Twin API has replaced the OpenNMS REST API for synchronizing Trapd configuration updates.
Additional settings are needed on both on Minion and {page-component-title} to enable Twin.

Refer to deployment section from the docs for details.
```

**Review by mmahacek — COMMENTED** [review]
Is this really just for Kafka, or does this apply to any non-embedded AMQ deployment?

**Comment by cgorantla** [comment]
> Is this really just for Kafka, or does this apply to any non-embedded AMQ deployment?

Added grpc to notes as well.

File: docs/modules/releasenotes/pages/whatsnew.adoc [hunk 2]
```diff
@@ -29,6 +29,14 @@ If you wish to revert to the previous behavior, create an `$OPENNMS_HOME/etc/ope
 RUNAS=root
 ----
 
+=== Additional config needed when using Kafka/Grpc for Minion Communication.
+
+The Twin API has replaced the OpenNMS REST API for synchronizing Trapd configuration updates.
+Additional settings are needed on both on Minion and {page-component-title} to enable Twin.
```
Comment (mmahacek) [inline docs/modules/releasenotes/pages/whatsnew.adoc:35]:
```suggestion
Additional settings are needed on both on Minion and {page-component-title} to enable Twin when not using the embedded ActiveMQ message broker.
```

File: docs/modules/releasenotes/pages/whatsnew.adoc [hunk 3]
```diff
@@ -29,6 +29,14 @@ If you wish to revert to the previous behavior, create an `$OPENNMS_HOME/etc/ope
 RUNAS=root
 ----
 
+=== Additional config needed when using Kafka/Grpc for Minion Communication.
```
Comment (Bonrob2) [inline docs/modules/releasenotes/pages/whatsnew.adoc:32]:
```suggestion
=== Additional configuration needed when using Kafka/gRPC for Minion communication.
```

**Review by Bonrob2 — CHANGES_REQUESTED** [review]
minor changes

**Review by mmahacek — APPROVED** [review]

**Review by Bonrob2 — APPROVED** [review]
LGTM!

**Review by RangerRick — APPROVED** [review]

**Pull request merged**